### PR TITLE
[docs] add Analytics

### PR DESCRIPTION
**Pull request by guillaumejparis**

### CODE REVIEW

File: docs/administration/parameters.md [hunk 1]
```diff
@@ -33,3 +33,13 @@ This option can be deactivated to have a permanent Announcement.
 ![Broadcast message display not dismissible](assets/broadcast_message_display_notdismissible.png)
 
 ⚠️ Only one Announcement is displayed at a time. Dismissible Announcement are displayed first, then the latest not dismissible Announcement.
+
+## Analytics
+
+!!! tip "Enterprise edition"
```
Comment (Goumies) [inline docs/administration/parameters.md:39]: @guillaumejparis what should be display with this line ?

Comment (guillaumejparis) [reply docs/administration/parameters.md:39]: image

File: docs/administration/parameters.md [hunk 2]
```diff
@@ -33,3 +33,13 @@ This option can be deactivated to have a permanent Announcement.
 ![Broadcast message display not dismissible](assets/broadcast_message_display_notdismissible.png)
 
 ⚠️ Only one Announcement is displayed at a time. Dismissible Announcement are displayed first, then the latest not dismissible Announcement.
```
[the remark this answers is not in the document]
Comment (labo-flg) [reply docs/administration/parameters.md:35]:
```suggestion
⚠️ Only one Announcement is displayed at a time. Dismissible Announcements are displayed first, then the latest not dismissible Announcement.
```

[the remark this answers is not in the document]
Comment (labo-flg) [reply docs/administration/parameters.md:35]: We might as well use suggestions for that  ;)

**Review by labo-flg — APPROVED** [review]

**Review by Archidoit — APPROVED** [review]

**Pull request merged**